### DISCUSSION TITLE
Add orderBy parameter support to getOrganizationList in OrganizationApi

### DISCUSSION THREAD

**Discussion by mario-jerkovic** [clerk/javascript#3152] [Feature Requests]
### Hello  👋

The `getOrganizationList` method in the `OrganizationApi` currently does not support the `orderBy` parameter. However, the [official Clerk backend API documentation](https://clerk.com/docs/reference/backend-api/tag/Organizations#operation/ListOrganizations) lists `order_by` as a supported parameter for listing organizations. This parameter is crucial for implementing sorted views of organizations directly from API calls.

### Expected Behavior
The `getOrganizationList` function in the `OrganizationApi.ts` file should allow developers to specify the `orderBy` parameter to sort the results based on specified fields as documented.

### Current Behavior
The current implementation of the `getOrganizationList` method in [`OrganizationApi.ts`](https://github.com/clerk/javascript/blob/1ac6f4b1ab602ba9390fbf4f132be4a76e554e35/packages/backend/src/api/endpoints/OrganizationApi.ts#L22) does not expose an `orderBy` parameter. The updated method signature could look like this:

```typescript
type GetOrganizationListParams = ClerkPaginationRequest<{
    includeMembersCount?: boolean;
    orderBy?: 'name' | 'members_count' | 'created_at' | '+name' | '+members_count' | '+created_at' | '-name' | '-members_count' | '-created_at'; 
    query?: string;
}>

class OrganizationAPI extends AbstractAPI {
    getOrganizationList(params?: GetOrganizationListParams): Promise<PaginatedResourceResponse<Organization[]>>;
}
```

**Comment by IGassmann** [comment]
Hey, @mario-jerkovic! I've opened a PR for it: https://github.com/clerk/javascript/pull/3164

**Reply by mario-jerkovic** [reply]
@IGassmann That was fast, thank you 🍻
